src/components/common/Modal.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export interface ModalProps {
+  isShowing: boolean;
+  message: string;
+  handleHide: React.MouseEventHandler;
+  handleConfirm: React.MouseEventHandler;
+  isFinishing?: boolean; // 투표 마감하기 버튼
+}
+
+const Modal = (props: ModalProps) => {
+  const { isShowing, message, handleHide, handleConfirm, isFinishing } = props;
+  return (
+    <>
+      {isShowing && (
+        <StModalWrapper>
+          <StModal>
+            <StModalContent>{message}</StModalContent>
+            <StModalSubContent isFinishing={isFinishing}>
+              *마감된 투표는 <span>라이브러리</span>에서 확인 가능해요!
+            </StModalSubContent>
+            <StButtonWrapper>
+              <button type="button" onClick={handleHide}>
+                취소
+              </button>
+              <button type="button" onClick={handleConfirm}>
+                확인
+              </button>
+            </StButtonWrapper>
+          </StModal>
+        </StModalWrapper>
+      )}
+    </>
+  );
+};
+
+export default Modal;
+
+const StModalWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
```

Comment:
이거 
```suggestion
position : fixed;
top : 0;
left: 0;
```
만 적어줘도 똑같은 효과 나올텐데!